public_suffix_list.dat

```diff
@@ -13781,6 +13781,11 @@ ie.ua
 // HostyHosting : https://hostyhosting.com
 hostyhosting.io
 
+// Hugging Face: https://huggingface.co
+// Submitted by Eliott Coyac <website@huggingace.co>
```

Comment:
There might be a typo in the email address ( `@huggingface.co` instead of `@huggingace.co`)?

Reply:
By the way, unrelated to this PR, just so you are aware, `huggingace.co` is currently a phishing site (leveraging the same typo) according to [VirusTotal](https://www.virustotal.com/gui/domain/huggingace.co)

Reply:
```suggestion
// Submitted by Eliott Coyac <website@huggingface.co>
```

Thanks for catching! :facepalm: 

(side-note: we did extend `hf.space` registration by 5 years today, will wait until it's propagated to whois to ping back)